papis/document.py

```diff
@@ -308,6 +308,9 @@ class Document(Dict[str, Any]):
 
         A :class:`DocHtmlEscaped` instance that can be used to escape keys
         in the document for use in HTML documents.
+
+    WARNING: running doc.save() without first running doc.set_folder()
+    will fail due to an unset info file path.
```

Comment:
This would be better added in the docs for `Document.save`. Ideally we'd also throw a nicer error in there (not sure what error it throws now?).

Reply:
It was `FileNotFoundError: [Errno 2] No such file or directory: ''`

I added a logger.error message and raised DocumentFolderNotFound.